### PR TITLE
Add OKRs to handbook

### PR DESCRIPTION
**Pull request by dcwalk**
Adding initial page to handbook to address https://github.com/hyphacoop/organizing/issues/173, where we say we want "handbook PR with process, Os, link to spreadsheet"

As we haven't fully settled on our process yet (see https://github.com/hyphacoop/organizing/issues/199), perhaps this is enough to start?

### CODE REVIEW

File: okrs.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+**Business Planning WG**:  
+O1. Build our engagement strategy  
+O2. Develop our leads and opportunities pipeline  
+O3. Gain a better sense of work that people want to do (cross-cut with Infra WG)  
+O4. Assess our financial sustainability (as “capacity to endure”) (cross-cut with Finance WG)  
+O5. Shorten our lead-to-proposal time (i.e. identifying an opportunity > getting member buy-in > submitting a proposal)  
+
+**Finance WG**:  
+O1. Define robust processes to support financial activities of the co-operative
```
Comment (benhylau) [inline okrs.md]:
```suggestion
O1. Define robust processes to support financial activities of the co-operative  
```

File: okrs.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Objectives and Key Results (OKRs)
+
+Hypha uses Objectives and Key Results for planning and setting collective goals.
+We currently define OKRs at the working group level and update them quarterly in our [**OKR tracking sheet**](https://docs.google.com/spreadsheets/d/1hMFS3IhzZOFQA-yjt7lFcoVO0Ry6erFEH7D1NswvDtA/edit). 
+
+## About OKRs
+
+_Adapted from [Trello blog](https://blog.trello.com/okrs-set-achieve-track-trello) shared by patcon in chat._
```
Comment (benhylau) [inline okrs.md]: A little too much information to reference individual sharing in chat for handbook.

```suggestion
_Adapted from [Trello blog](https://blog.trello.com/okrs-set-achieve-track-trello)._
```

**Comment by benhylau** [comment]
@dcwalk apply my comment and merge?

**Comment by dcwalk** [comment]
Thanks for review @benhylau!

File: okrs.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+O5. Shorten our lead-to-proposal time (i.e. identifying an opportunity > getting member buy-in > submitting a proposal)  
+
+**Finance WG**:  
+O1. Define robust processes to support financial activities of the co-operative  
+O2. Have well-defined roles and responsibilities to ensure smooth financial activities  
+O3. Effectively communicate our financial outlook to members  
+
+**Governance WG**:  
+O1. Formalize a governance structure that meets legal requirements  
+O2. Build [governance] relationships with "neighbours" (cross-cut with Biz Dev/Ops WGs)
```
Comment (uditvira) [inline okrs.md]: Nitpick: In other places we call the business wg "Business Planning" and not "Biz Dev"

Comment (dcwalk) [reply okrs.md]: Yes I've noticed this, however we also use BizDev a bunch of places as a shorthand (e.g., calendar). Happy to change here

**Pull request merged**